ansible/roles/couchdb/tasks/deploy.yml

```diff
@@ -21,25 +21,39 @@
     volume_dir: "{{ instance.volume.fsmount | default( '/mnt/' + group_names|first, true ) }}:/usr/local/var/lib/couchdb"
   when: (block_device is defined) and (block_device in disk_status.stdout)
 
-- name: "pull the klaemo/couchdb:{{ couchdb.version }} image"
-  shell: "docker pull klaemo/couchdb:{{ couchdb.version }}"
+- name: "set the num_instances"
+  vars:
+    instance: "{{instances | selectattr('name', 'equalto', 'db') | list }}"
+  set_fact:
+    num_instances: "{{ instance.num_instances | default(1) }}"
+  when: instances is defined
+
+- name: "set the num_instances"
+  set_fact:
+    num_instances: "1"
+  when: instances is undefined
+
+- name: "pull the apache/couchdb:{{ couchdb.version }} image"
+  shell: "docker pull apache/couchdb:{{ couchdb.version }}"
   retries: "{{ docker.pull.retries }}"
   delay: "{{ docker.pull.delay }}"
 
 - name: (re)start CouchDB
   docker_container:
     name: couchdb
-    image: klaemo/couchdb:{{ couchdb.version }}
+    image: apache/couchdb:{{ couchdb.version }}
     state: started
     recreate: true
     restart_policy: "{{ docker.restart.policy }}"
     volumes: "{{volume_dir | default([])}}"
     ports:
       - "{{ db_port }}:5984"
+      - "4369:4369"
+      - "9100:9100"
```

Comment:
I want to discuss this further.
Please refer to this: http://docs.couchdb.org/en/latest/cluster/setup.html#firewall

Those two ports are used when Erlang VM finds other nodes.
APAIK, 4369 is reserved to find other nodes, and 9100 is reserved to use for communication among nodes.

Normally, erlang VM uses random ports between 9100~9200, but it is forced to use 9100 only in `apache/couchdb`.

Regarding 9100, I think we can change it, but we need to update `apache/couchdb` image.
That should be different on every nodes, I am not sure it works in that situation.
You can refer to this: https://github.com/apache/couchdb-docker/blob/master/2.1.0/vm.args#L14

And with regard to 4369, since it is reserved at Erlang VM layer, I am not sure we can change it.

@cbickel Do you have any idea?